src/jit/SimdArm64Inl.h

```diff
@@ -198,6 +203,23 @@ static void simdEmitI64x2AllTrue(sljit_compiler* compiler, sljit_s32 rd, sljit_s
     sljit_emit_op_flags(compiler, SLJIT_MOV, rd, 0, SLJIT_ORDERED_EQUAL);
 }
 
+static void simdEmitDot(sljit_compiler* compiler, sljit_s32 rd, sljit_s32 rn, sljit_s32 rm)
+{
+#ifdef __ARM_FEATURE_DOTPROD
+    simdEmitOp(compiler, SimdOp::sdot | SimdOp::S4, rd, rn, rm);
+#else
+    auto tmpReg = SLJIT_FR2;
+
+    simdEmitOp(compiler, SimdOp::smull | SimdOp::H8, tmpReg, rn, rm); // tmpReg = rn * rm lower
```

Comment:
Use full sentences for comments. Rm is also a tmpReg?

Reply:
Rm is one of the input register and it is used as a tmp should i use an other register for this?